### DISCUSSION TITLE
unable to send  the test email

### DISCUSSION THREAD

**Discussion by babathota** [PHPMailer/PHPMailer#3088] [Q&A]
hi i am geeting error Error: SMTP Error: Could not connect to SMTP host. Connection failed. stream_socket_enable_crypto(): SSL operation failed with code 1. OpenSSL Error messages:
error:1416F086:SSL routines:tls_process_server_certificate:certificate verify failed

**Comment by Synchro** [comment]
Have you considered [reading the docs on this exact subject](https://github.com/PHPMailer/PHPMailer/wiki/Troubleshooting#certificate-verification-failure)?
